src/data_handling/data_interface.py

```diff
@@ -0,0 +1,15 @@
+class DataInterface:
+    """
+    Class for storing data created in DataHandler.
+    """
+    def __init__(self):
```

Comment:
Legyen egy `data` argumentum, ami egy `dict`. A konstruktorban egyelőre maradjon a `None`-nal inicializálás, de legyen egy `if data is not None` feltétel: ha ez igaz, akkor a korábbi projektben látott módon töltsük fel a tagváltozókat az átadott adatokkal (azt feltételezve, hogy a `data` kulcsai ugyanazok, mint a fenti tagváltozók):
```
for key, value in data.items():
            setattr(self, key, value)
```
Írjuk le a konstruktorba, hogy milyen kulcsneveket várunk a `data` dictionary-ben.